lib/notifier.rb

```diff
@@ -19,6 +19,8 @@ def initialize(notification, pages_url, slack_url, live, limit = -1)
   def run
     payloads = message_payloads(pages_per_channel)
 
+    return if payloads.nil?
```

Comment:
Doesn't look like `message_payloads()` can return `nil` to me - it returns `grouped_pages.map` which will always be an enumerable (possibly `[]` in the worst case).

Perhaps:

```suggestion
    return if payloads.empty?
```

would be better?